### PR TITLE
Update README for new args

### PR DESCRIPTION
**Pull request by black-dragon74**
Discussed here: https://issues.redhat.com/browse/OCSDOCS-1448

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,8 +13,9 @@ oc adm must-gather --image=registry.redhat.io/odf4/odf-must-gather-rhel9:v4.13 -
 
 List of arguments that can be passed to the odf-must-gather are:
 ```
+-o,  --odf                Collect ODF logs
 -d,  --dr                 Collect DR logs
--pc,  --provider          Collect openshift-storage-client logs from a provider/consumer cluster
+-pc, --provider           Collect openshift-storage-client logs from a provider/consumer cluster
```
Comment (yati1998) [inline README.md:18]: As discussed with Neha, we can remove this for now. We won't be documenting for provider/consumer this release. So once it is GAd and released, we will add this in documentations.

Comment (black-dragon74) [reply README.md:18]: Alright :)

File: README.md [hunk 2]
```diff
@@ -15,7 +15,6 @@ List of arguments that can be passed to the odf-must-gather are:
 ```
 -o,  --odf                Collect ODF logs
```
Comment (yati1998) [inline README.md:16]: Collect ODF logs (includes content from arguments n,c,cl,cs contents)

Comment (black-dragon74) [reply README.md:16]: Or, shall we write it as (includes everything except DR)?

Comment (black-dragon74) [reply README.md:16]: Right now, it makes no point to use `-o` in conjunction with anything other than `-dr`.

Comment (yati1998) [reply README.md:16]: we will have to update it again after we include other parameters, so better to clearly mention the contents.

Comment (black-dragon74) [reply README.md:16]: Ack

**Review by yati1998 — APPROVED** [review]

**Pull request merged**